torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -367,30 +367,33 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                 node_users = list(new_node.users.keys())
                 for node_user in node_users:
                     node_user.replace_all_uses_with(new_node)
```

Comment:
In this particular case, we need a manual RAUW loop.  This piece of code remains the same.